lib/ssz_ex.ex

```diff
@@ -398,4 +404,16 @@ defmodule LambdaEthereumConsensus.SszEx do
     |> Enum.map(fn {_, schema} -> variable_size?(schema) end)
     |> Enum.any?()
   end
+
+  defp pack(value, size) when is_integer(value) and value >= 0 do
+    pad = 256 - size
```

Comment:
We should name the `256`. Maybe `@leaf_size`. Wdyt?

Reply:
Yeah. Maybe `@max_bit_size`?

Reply:
The value per byte is called `BYTES_PER_CHUNK` in the spec, so how about `@bits_per_chunk`?

Reply:
Changed to this. It's better